### PR TITLE
[WIP] small opportunistic clarity edits

### PR DESCRIPTION
**Pull request by li3n3**
WIP for now; I'm just fixing little things as I come across them.

Deeply tempted to regularize line wrap length (some are pretty tidy; some are very very very long), but I've held off for now.

### CODE REVIEW

**Comment by felicianotech** [comment]
What I try to do, bringing this habit from the dev world, is line-wrapping at 80 characters.

**Comment by ndintenfass** [comment]
FWIW, I hate 80-char line wrapping in prose.

**Comment by felicianotech** [comment]
Personally I like 100 char but I've gotten weird looks in the past about using that many. :)

In a GUI text editor, soft word wrapping is fine but I know when I'm in Vim, having a set width is VERY nice.

**Comment by li3n3** [comment]
Vim is exactly why I care more than 0% — it dramatically simplifies text maneuvering, and Markdown chomps it all up anyway. (Learning about `gq` was like getting a surprise present.)

File: README.md [hunk 1]
```diff
@@ -73,7 +74,7 @@ jekyll serve
 ```
 
 Jekyll will build the site and start a web server. It can be viewed in your 
-browser at [](http://localhost:4000/docs/). To stop Jekyll and regain controll 
+browser at <http://localhost:4000/docs/>. To stop Jekyll and regain control
```
Comment (smart-alek) [inline README.md]: I would cut "and regain control of your terminal".

File: jekyll/_docs/troubleshooting-guide.md [hunk 2]
```diff
@@ -47,8 +46,8 @@ compilation of the ones we see most frequently:
   suspect this, it might be a good idea to check the status pages of the
   services you rely on.
 
-### None of these is the reason?
-You could SSH into your build container to debug interactively,
+### If none of these is the reason
```
Comment (smart-alek) [inline jekyll/_docs/troubleshooting-guide.md]: Replace to "### Otherwise / Else" (just something less wordy)

**Comment by smart-alek** [comment]
Also, I'm with @ndintenfass on the char limits in prose. Char limits make reviews harder, and you can turn on text wrapping in your editor of choice if a line gets too long.

File: README.md (hunk 1 above)
Comment (li3n3) [inline README.md]: And here I was gonna stop with just correcting spelling! Will do.

File: jekyll/_docs/troubleshooting-guide.md (hunk 2 above)
Comment (li3n3) [inline jekyll/_docs/troubleshooting-guide.md]: I dig it. Will take a scythe to it.

**Pull request merged**